packages/wouter/types/index.d.ts

```diff
@@ -113,7 +114,7 @@ export function Redirect<H extends BaseLocationHook = LocationHook>(
 ): ReactElement<any, any> | null;
 
 export function Link<H extends BaseLocationHook = LocationHook>(
-  props: PropsWithChildren<LinkProps<H>>,
+  props: PropsWithChildren<LinkProps<H>> & RefAttributes<HTMLAnchorElement>,
```

Comment:
My guess is, you can try to infer the type of `children` prop and then use `ElementRef<props["children"]>`

Reply:
it doesn't work

and it makes no sense to pass ref object to children:

```jsx
<Link ref={ref}>
  <MyCustomLinkComponent />
</Link>
```

should be 👇🏻👇🏻👇🏻
```jsx
<Link>
  <MyCustomLinkComponent ref={ref} />
</Link>
```

in userland